packages/expo-notifications/src/NotificationCategoriesModule.ts

```diff
@@ -11,7 +11,7 @@ export default {
   ): Promise<null> {
```

Comment:
```suggestion
  ): Promise<NotificationCategory | null> {
```

(also make change to `NotificationCategoriesModule.types.ts`)